roles/example-cnf-catalog/tasks/main.yml

```diff
@@ -68,11 +68,16 @@
   until: pkg_manifest_trex.resources|length != 0
   failed_when: pkg_manifest_trex.resources|length == 0
 
-- name: create namespace
+- name: Create namespace
```

Comment:
@tkrishtop @tonyskapunk 
I think it is safe to remove the namespace creation from here. This example-cnf-catalog role is a custom implementation for example-cnf and is dependent on what we are doing in the hook.

I think creating the NS with the proper labels during pre-run as it is been defined in https://github.com/dci-labs/example-cnf-config/pull/60/ is ok. And we can separate the scope of both changes.

Reply:
@betoredhat also consider this as a standalone tool and not always run with dci labs scripts.

Reply:
@krsacme  true, thanks for checking. let's update the 2 places. Thanks